README.md

```diff
@@ -24,7 +24,7 @@ and build system unreliable.
 When it comes to building portable ELF binaries, the libc plays an important role. The linker will
 try to link against new symbols, and since the GNU libc has symbol versioning, linking against a
 newer glibc will prevent that program from running on a system using an older glibc. To solve this,
-we ship glibc 2.26 with the sysroots, which should be old enough by now to make all programs
+we ship glibc 2.37 with the sysroots, which should be old enough by now to make all programs
```

Comment:
2.37 is not old enough. We need to keep it 2.26 for a while. This is the most important feature of this toolchain.